ZappAppConnector/Protocols/ZAAppDelegateConnectorChromecastProtocol.h

```diff
@@ -0,0 +1,15 @@
+//
+//  ZAAppDelegateConnectorChromecastProtocol.h
+//  ZappAppConnector
+//
+//  Created by Avi Levin on 21/12/2017.
+//  Copyright © 2017 Applicaster Ltd. All rights reserved.
+//
+@protocol ZAAppDelegateConnectorChromecastProtocol
```

Comment:
Please add nullability annotations (or just write it in Swift).

Reply:
This protocol is used to connect between 2 obj c sides. 
I added NS_ASSUME_NONNULL_BEGIN and NS_ASSUME_NONNULL_END anyway.

Let talk more about it on Sunday/Monday.